sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/GenericResourceContainer.cs

```diff
@@ -75,8 +81,14 @@ public override Response<GenericResource> Get(string resourceId, CancellationTok
             }
         }
 
-        /// <inheritdoc/>
-        public override async Task<Response<GenericResource>> GetAsync(string resourceId, CancellationToken cancellationToken = default)
+        /// <summary>
+        /// Gets details for this resource from the service.
+        /// </summary>
+        /// <param name="resourceId"> The ID of the resource to get. </param>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="CancellationToken.None" />. </param>
+        /// <returns> A <see cref="Task"/> that on completion returns a response with the <see cref="Response{GenericResource}"/> operation for this resource. </returns>
+        /// <exception cref="ArgumentException"> resourceId cannot be null or a whitespace. </exception>
+        public virtual async Task<Response<GenericResource>> GetAsync(string resourceId, CancellationToken cancellationToken = default)
```

Comment:
If this is not virtual I get `System.InvalidOperationException: Client type contains public non-virtual async method GetAsync`